.mvn/wrapper/maven-wrapper.properties

```diff
@@ -15,4 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.5/apache-maven-3.9.5-bin.zip
+distributionSha256Sum=7822eb593d29558d8edf87845a2c47e36e2a89d17a84cd2390824633214ed423
```

Comment:
This is apparently not affected by MWRAPPER-103 because it validated in Java code (probably [here](https://github.com/apache/maven-wrapper/blob/maven-wrapper-3.2.0/maven-wrapper/src/main/java/org/apache/maven/wrapper/Installer.java#L94-L98))

Reply:
Oh, neat, so maybe there's no need for `sha256sum` to be present on the machine, either?

Reply:
Oh, oops, we would need it if we were verifying the wrapper. (I'd gotten myself mixed up again between `script` and `only-script`.) So it's true that we don't need it now, but we'd like to verify the wrapper someday, at which point we will need it.